docs/plugins/hooks.rst

```diff
@@ -922,3 +922,35 @@ octoprint.ui.web.templatetypes
    :param dict template_sorting: read-only dictionary of currently configured template sorting specifications
    :return: a list of 3-tuples (template type, rule, sorting spec)
    :rtype: list
+
+.. _sec-plugins-hook-timelapse-capture-pre:
+
+octoprint.timelapse.capture.pre
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Might sound weird but: could you fix that underline to match the length of the headline? Sphinx/RST is a bit whiny when it comes to stuff like that. Same with the post hook.